examples/widgets/media_query.dart

```diff
@@ -72,13 +72,14 @@ class MediaQueryExample extends StatelessComponent {
       items.add(new AdaptiveItem("Item $i"));
 
     if (MediaQuery.of(context).size.width < _gridViewBreakpoint) {
-      return new Block(
-        items.map((AdaptiveItem item) => item.toListItem()).toList()
+      return new ScrollableList2(
+        itemExtent: 50.0,
```

Comment:
This example is switching from using a non-viewporting widget (Block) to a ScrollableList, which requires you to specify the itemExtent.  That's the same as it would be with the original ScrollableList.